proto/net.proto

```diff
@@ -46,11 +46,21 @@ message Weights {
     optional Layer biases = 2;
     optional Layer bn_means = 3;
     optional Layer bn_stddivs = 4;
+    optional Layer bn_gammas = 5;
+    optional Layer bn_betas = 6;
+  }
+
+  message SEunit {
+    optional Layer w1 = 1;
+    optional Layer b1 = 2;
+    optional Layer w2 = 3;
+    optional Layer b2 = 4;
   }
 
   message Residual {
     optional ConvBlock conv1 = 1;
     optional ConvBlock conv2 = 2;
+    optional SEunit se = 3;
```

Comment:
Maybe some minimal comments? `se`, `w1`, `b2` are very short acronyms and it will be hard to understand what relates to what when we extend the proto further.